src/styles/main.scss

```diff
@@ -1,3 +1,51 @@
+$height: 300px;
+
 body {
   margin: 0;
 }
+
+.blocks {
+  @media (min-width: 600px) {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 900px) {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
```

Comment:
if you use `display: grid ` for the whole  `.blocks`  you can get away with having just one `@media`

Reply:
With my current knowledge I don't think I can hack it

Reply:
Just take the `display:grid` out of the media query. This way it will be automatically applied to the `.blocks` block on all sceen sizes. In media queries leave only the things that should change.

Like this

blocks {
display: grid;

  @media (min-width: 600px) {
    grid-template-columns: repeat(2, 1fr);
  }

@media (min-width: 900px) {
    grid-template-columns: repeat(3, 1fr);
  }
}